### PR TITLE
Update PHP-CS-Fixer

### PR DESCRIPTION
**Pull request by klees**
Hi everyone,

I only wanted to changes stuff in [Services/Component](https://github.com/ILIAS-eLearning/ILIAS/pull/3901), turns out that there is some yak shaving hidden there.

[PR 3901](https://github.com/ILIAS-eLearning/ILIAS/pull/3901) fails, because PHP-CS-Fixer does not support PHP 8.1. An update of PHP-CS-Fixer required a major update of `symfony/console`, which in turn required an update of the dependencies of simplesamlphp.

This is all contained in here.

@mjansenDatabay Could you have a look here and merge this if ok?

Thank!

### CODE REVIEW

**Comment by klees** [comment]
The yak seems to be naked now.

**Comment by mjansenDatabay** [comment]
Hmmm, I wonder why the builds pass in `trunk`, but they fail for PRs.

**Comment by klees** [comment]
image

this would be the explanation...

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,11 @@
 	"authors": [],
 	"config": {
 		"optimize-autoloader": true,
-		"vendor-dir": "./libs/composer/vendor"
+		"vendor-dir": "./libs/composer/vendor",
+		"allow-plugins": {
+			"simplesamlphp/composer-module-installer": true,
```
Comment (mjansenDatabay) [inline composer.json:10]: I like! +1 for the usage of additional security layers.

**Review by mjansenDatabay — APPROVED** [review]

**Pull request merged**